spec/models/bulk_update/trainee_upload_spec.rb

```diff
@@ -5,4 +5,15 @@
 RSpec.describe BulkUpdate::TraineeUpload do
   it { is_expected.to belong_to(:provider) }
   it { is_expected.to have_many(:trainee_upload_rows).dependent(:destroy) }
+
+  it do
+    expect(subject).to define_enum_for(:status).with_values(
+      pending: "pending",
+      validated: "validated",
+      in_progress: "in_progress",
+      succeeded: "succeeded",
+      failed: "failed",
+      cancelled: "cancelled",
```

Comment:
Just wondering do we need a clean up job or something?

Something like?

https://github.com/DFE-Digital/register-trainee-teachers/blob/main/config/sidekiq_cron_schedule.yml#L24-L27

Reply:
Will be implemented as part of [7850-create-status-of-new-trainee-files-page](https://trello.com/c/MNr8s1Cc/7850-create-status-of-new-trainee-files-page)